mediapipe/tasks/ios/test/vision/image_segmenter/MPPImageSegmenterTests.mm

```diff
@@ -468,6 +468,45 @@ - (void)imageSegmenter:(MPPImageSegmenter *)imageSegmenter
   }
```

Comment:
Can this method be moved to "Assert Segmenter Results"?

Reply:
I think this method should be moved out of the section that contains the test methods. I will move it on import.